faucet/generated-wallet-details.json

```diff
@@ -0,0 +1 @@
+{"wallet-id":"Ae2tdPwUPEZJkLTchA9mETtyXumBT1K1gqtX3dgXUguJFLuvyNU9BzQXrEK","recovery-words":["relax","secret","seek","story","supreme","meat","judge","onion","weasel","sponsor","garlic","erase"],"address":"DdzFFzCqrhswmBoqDubnsZZpvnfFbjgwLwFyk4vC5jJJf75m36f4M3f67k4iia1RQdCUPmJHV7c9TK2ogfzjoQSYHKCoM4YTv6zrmr28","account-index":2147483648}
```

Comment:
Is it worth pretty printing that JSON so its a bit more human readable?

Reply:
That wasn't actually supposed to be in there. I've added it to ignore.